lib/cssom.js

```diff
@@ -11,12 +11,22 @@
 declare class StyleSheet {
   disabled: boolean;
   href: string;
+  media: MediaList;
   ownerNode: Node;
   parentStyleSheet: ?StyleSheet;
   title: string;
   type: string;
 }
 
+declare class StyleSheetList {
+    length: number;
+    [index: number]: StyleSheet;
+}
+
+declare class MediaList {
+
```

Comment:
wow, i've never seen MDN missing anything!

looks like the spec is here: http://www.w3.org/TR/DOM-Level-2-Style/stylesheets.html#StyleSheets-MediaList

you don't have to keep going down a deeper and deeper rabbit hole, though. if you want to leave it empty for now that's fine and we can come back to it later.